test/byContract/IexecAccessors/IexecAccessorsABILegacy.test.ts

```diff
@@ -2,8 +2,8 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import { AddressZero } from '@ethersproject/constants';
```

Comment:
```suggestion
import { ZeroAddress } from 'ethers';
```
and update line 179.

Reply:
Sure see [Import from ethers](https://github.com/iExecBlockchainComputing/PoCo/pull/189/commits/01a96ab3e20eece28b60f32d3a08f1f93304bd37) (same comment everywhere)